MediaBrowser.Model/Dlna/StreamBuilder.cs

```diff
@@ -395,7 +385,7 @@ private StreamInfo BuildAudioItem(MediaSourceInfo item, AudioOptions options)
             // If device requirements are satisfied then allow both direct stream and direct play
             if (item.SupportsDirectPlay)
             {
-                if (IsItemBitrateEligibleForDirectPlay(item, GetBitrateForDirectPlayCheck(item, options, true) ?? 0, PlayMethod.DirectPlay))
+                if (IsItemBitrateEligibleForDirectPlay(item, options.GetMaxBitrate(false) ?? 0, PlayMethod.DirectPlay))
```

Comment:
```suggestion
                if (IsItemBitrateEligibleForDirectPlay(item, options.GetMaxBitrate(true) ?? 0, PlayMethod.DirectPlay))
```
because this is audio (also `true` as the last argument).
But I never got any other value than `MaxBitrate`, i.e. `MaxStaticBitrate` or `MaxStaticMusicBitrate` :man_shrugging: 

Fun fact, the other `GetBitrateForDirectPlayCheck` call also has `true` as the last argument.

Reply:
Yep that was my copy&paste error, good catch